source/hardware/controllers/reloop_ready.rst

```diff
@@ -0,0 +1,92 @@
+.. _reloop-ready:
+
+Reloop Ready
+=============
+
+.. sectionauthor::
+   Nikolaus Einhauser <nikolaus.einhauser@mixxx.org>
+
+The Reloop Ready controller is an entry-level 2-Deck :term:`MIDI` controller
+featuring a built-in 4-channel USB audio interface.
+
+-  `Manufacturer’s product page <https://www.reloop.com/reloop-ready>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/reloop-ready>`__
+-  `Instruction Manual <https://www.reloop.com/media/catalog/product/pdf/2/4/3/243598_Reloop_IM.pdf>`__
+
+..
+    -  `Mapping Forum Thread <>`__ TODO
```

Comment:
Do we have a forums thread?

Reply:
Not yet. I didn't bother 'packaging" the mapping for the forums yet.